src/Item/ViewItem/sections/ItemData/ItemData.js

```diff
@@ -33,9 +32,7 @@ const ItemData = ({ itemData, refLookup, referenceTables }) => {
       </Row>
       <Row>
         <Col xs={12}>
-          <Headline>
-            <FormattedMessage id="ui-inventory.itemCallNumber" />
-          </Headline>
+          <h3><FormattedMessage id="ui-inventory.primaryItemCallNumber" /></h3>
```

Comment:
I'm not against changing some of these tags to `<h3>`, but I think we can still use `<Headline tag="h3">` component as it may provide some default stripes styling to be consistent with other modules

Reply:
Thanks @BogdanDenis !
Well, using` Headline tag="h3"` will reduce the space above the Headline and the font-size will become smaller, so the small visual effect of the headline will get lost completely:
<img width="350" alt="image" src="https://github.com/user-attachments/assets/3c867ab0-511d-477d-8e4f-5a7eceb2f9e3" />

If you compare with `h3`:
<img width="350" alt="image" src="https://github.com/user-attachments/assets/0412f73e-427e-430d-bf42-e105a604a5c9" />

Reply:
Headline tag="h3" - font-size: 14px
h3 - font-size: 16.38px
KeyValue - font-size: 14.7px

Reply:
I _am_ against changing `<headline>` to `<h3>` for all the reasons @BogdanDenis identifies. Use `tag="h3" size="medium"` (or `large`, `x-large`, etc; see [the storybook for details](https://dev.folio.org/stripes-components/?path=/story/components-headline--basic-usage)). I'll start a conversation with @JohnC-80 about if/how we can sync the h1 to h6 tags to headline's small to xx-large sizes.

Reply:
Sorry, can't see any advantage of using headline. The styling of the headlines should be adapted.

Reply:
changed `h3` to `Headline size="large" tag="h3"`